src/domain/hanke/hankeView/HankeView.tsx

```diff
@@ -123,6 +125,140 @@ const HankeAreaInfo: React.FC<AreaProps> = ({ area, index }) => {
   );
 };
 
+const HaittojenhallintasuunnitelmaInfo: React.FC<AreaProps> = ({ area, index }) => {
+  const { t } = useTranslation();
+  const locale = useLocale();
+
+  return (
+    <Accordion
+      language={locale}
+      heading={area.nimi || t('hanke:alue:title', { index: index + 1 })}
+      initiallyOpen
+      className={styles.haittojenhallintasuunnitelmaContainer}
+    >
+      <div>
+        <FormSummarySection className={styles.odd}>
+          <SectionItemTitle>
+            {t('hankeForm:haittojenHallintaForm:nuisanceType:YLEINEN')}
+          </SectionItemTitle>
+          <SectionItemContent>
+            <Box pr="calc(var(--spacing-l) * 3)">
+              <Flex alignItems="flex-start" justifyContent="space-between">
```

Comment:
This Flex is probably not needed?

Reply:
You're right! I had some trouble getting all the sections aligned the same way regardless if they have <HaittaIndex> or not and this was left from some of those experiments. I removed this <Flex> from here and from MUUT section as well.